entity-service/src/main/kotlin/com/egm/stellio/entity/repository/PropertyRepository.kt

```diff
@@ -1,8 +1,34 @@
 package com.egm.stellio.entity.repository
 
 import com.egm.stellio.entity.model.Property
+import org.springframework.data.neo4j.annotation.Query
 import org.springframework.data.neo4j.repository.Neo4jRepository
 import org.springframework.stereotype.Repository
 
 @Repository
-interface PropertyRepository : Neo4jRepository<Property, String>
+interface PropertyRepository : Neo4jRepository<Property, String> {
+    @Query(
+        "MATCH (property:Property { id: \$id }) " +
+            "RETURN property " +
+            "LIMIT 1"
+    )
+    fun getPropertyCoreById(id: String): Property?
```

Comment:
I checked one or two, they seem to be used only in unit tests?

Reply:
yes, since the findById throws cast exception

Reply:
updated [here](https://github.com/stellio-hub/stellio-context-broker/pull/191/commits/2a36d4cb39039c54c5cb167dfc68d3455b40fbd4), can you please check the location/bname of added interfaces ?

Reply:
This is a major problem if we can no longer use the findById (and similar) built-in functions. 

And, btw, the spec says nothing about ids of props and relationships (we use ids internally but that's not NGSI-LD conformance).